components/entity-access-rules/rest/src/main/java/org/phenotips/data/permissions/rest/internal/NameAndEmailExtractor.java

```diff
@@ -109,7 +115,7 @@
         if (groupObject != null) {
             email = groupObject.getStringValue("contact");
         }
-        String name = document.getTitle();
+        String name = document.getRenderedTitle(Syntax.PLAIN_1_0, this.xcontextProvider.get());
```

Comment:
TBH I would have done something like this, to avoid introducing a dependency on XWikiContext:
```suggestion
        String name = StringUtils.isBlank(document.getTtile()) ? : document.getName() : document.getTitle()
```
(not tested)